packages/components/modal/src/components/osds-modal/osds-modal.tsx

```diff
@@ -43,16 +43,21 @@ export class OsdsModal implements OdsModalAttribute, OdsModalMethod {
   @Method()
   async open(): Promise<void> {
     this.masked = false;
+    this.inertBodyChildren(this.masked);
```

Comment:
changing the `masked` Prop value will trigger the `masked` Watch, so the inert change should only be executed once (in the Watch handler)

Reply:
Problem was ["The `@Watch()` decorator does not fire when a component initially loads."](https://stenciljs.com/docs/reactive-data#the-watch-decorator-watch).